server/db/models/category.js

```diff
@@ -9,4 +10,15 @@ const Category = db.define('category', {
   }
 })
 
+Category.getProducts = async id => {
```

Comment:
Ah I see, this is more `getWithProducts` than `getProducts`.

I would consider using a Sequelize Scope instead of a static method for this eager loading: http://docs.sequelizejs.com/manual/tutorial/scopes.html

Reply:
…PS – though a static method is fine / valid, I am just using this as an excuse to teach you about Sequelize Scopes.